backend/canisters/user_index/impl/src/lib.rs

```diff
@@ -207,6 +214,33 @@ impl Data {
             }
         }
     }
+
+    fn register_super_admin_bot(&mut self, canister_id: CanisterId, username: String, now: TimestampMillis) {
```

Comment:
Why doesn't this need to be public?

Reply:
It's in the root and modules can access function and types in their ancestors.
So things defined in `lib` only need to be `pub` if they're accessed from other packages.